scripts/setup-appstore-api.sh

```
@@ -0,0 +1,127 @@
+    else
+        echo "❌ Failed to generate private key"
+        exit 1
+    fi
+}
+
+# Set up GitHub secrets
+setup_github_secrets() {
+    echo "🔒 Setting up GitHub secrets..."
+    
+    # Check if authenticated with GitHub CLI
+    if ! gh auth status &> /dev/null; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)